dev/js/pages/UserLoginView/index.js

```diff
@@ -26,29 +29,56 @@ const LoginView = () => {
 
   const loginSubmitHnadler = (event) => {
     event.preventDefault();
+    if (email === '' || password === '') {
+      return;
+    }
     userLogin({ email, password }, dispatch, (err, data) => {
       if (data) {
-        console.log('data', data);
+        setEmail('');
+        setPassword('');
+        window.location.href = 'http://' + window.location.host + '/dashboard';
```

Comment:
is this the right approach ? lets not hardcode things..
just send to '/dashboard', it will figure out protocol and domain on its own